app/v2/partials/directives/proposals-ideas-searchbox.html

```diff
@@ -124,6 +124,9 @@ <h4 class="widget__label">
       <li class="sarch_mode__popularity" ng-click="setSorting('popularity')" ng-class="{active: sortingIs('popularity')}">
         <a href="">{{'Popularity' | translate}}</a>
       </li>
+      <li class="sarch_mode__popularity" ng-click="setSorting('most_commented')" ng-class="{active: sortingIs('most_commented')}">
```

Comment:
Notice that you need to add `most_commented_public` and `most_commented_members` in the backend to support this.